### PR TITLE
Updates table row for Cancel Note

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* /bcgov/entity#16927

*Description of changes:*
- Added type of note being cancelled.
image

- Modified note items to title case using new format utility to match design.
image

*Note for reviewers:*

I will add some tests before merging.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1513-c36r3gmx.web.app

**Review by doug-lovett — APPROVED** [review]
Last one from me. Looks great.

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 1]
```diff
@@ -84,7 +84,10 @@
         {{ getRegistrationType(item.registrationType) }}
         <span v-if="isPpr && !isChild"> - Base Registration</span>
       </div>
-      <div v-else class="pr-2">{{ item.registrationDescription }}
+      <div v-else class="pr-2">{{ multipleWordsToTitleCase(item.registrationDescription, true) }}
```
Comment (dimak1) [inline ppr-ui/src/components/tables/common/TableRow.vue:88]: Why is there a need to capitalize the sentence? Was it a requirement? The ticket didn't say much.

Comment (orelbn) [reply ppr-ui/src/components/tables/common/TableRow.vue:88]: It matches all of the designs and I confirmed with Tracey that they intended it to be in title case.

Designs:
image
image
image

Comment (dimak1) [reply ppr-ui/src/components/tables/common/TableRow.vue:88]: I think it should be added in content, instead of making title cases.

Comment (orelbn) [reply ppr-ui/src/components/tables/common/TableRow.vue:88]: What do you mean by added in content?

Comment (dimak1) [reply ppr-ui/src/components/tables/common/TableRow.vue:88]: The text should come from content/resources as it without manipulating title cases.

Comment (dimak1) [reply ppr-ui/src/components/tables/common/TableRow.vue:88]: Resolving to move ahead.

**Review by dimak1 — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1513-c36r3gmx.web.app

**Pull request merged**